### PR TITLE
collecting data from base_coverage is optional

### PR DESCRIPTION
**Pull request by GatKrieger**
for the terra PR: gat/add_ssqc_to_trim_align_sort

### CODE REVIEW

File: ugvc/scripts/sorter_to_h5.py [hunk 1]
```diff
@@ -63,51 +63,56 @@ def sorter_to_h5(
             h5_dict[row["H5_key"]][row["H5_item"]] = json_data[row["key"]]
 
     # extract coverage stats
-    df_coverage_histogram = (
-        pd.concat(
-            (pd.DataFrame(json_data["base_coverage"][key], columns=[key]) for key in json_data["base_coverage"].keys()),
-            axis=1,
+    if not json_data["base_coverage"] == {}:  # if base_coverage is not empty
```
Comment (kenissur) [inline ugvc/scripts/sorter_to_h5.py]: To be on the safe side I would do
`if "base_coverage" in json_data and len(json_data["base_coverage"]) == 0`

I wouldn't rely on the fact that Ohad creates an empty field

File: ugvc/scripts/sorter_to_h5.py [hunk 2]
```diff
@@ -63,51 +63,56 @@ def sorter_to_h5(
...
+        # add the F80, F90, F95 metrics to RawWgsMetrics
+        h5_dict["RawWgsMetrics"]["FOLD_80_BASE_PENALTY"] = (
+            df_percentiles.loc["Q50", "Genome"] / df_percentiles.loc["Q20", "Genome"]
+        )
+        h5_dict["RawWgsMetrics"]["FOLD_90_BASE_PENALTY"] = (
+            df_percentiles.loc["Q50", "Genome"] / df_percentiles.loc["Q10", "Genome"]
+        )
+        h5_dict["RawWgsMetrics"]["FOLD_95_BASE_PENALTY"] = (
+            df_percentiles.loc["Q50", "Genome"] / df_percentiles.loc["Q5", "Genome"]
+        )
```
Comment (kenissur) [inline ugvc/scripts/sorter_to_h5.py:115]: I assume none of this has changed, right? It looks weird on GitHub

**Review by kenissur — APPROVED** [review]
Just one minor suggestion

File: ugvc/scripts/sorter_to_h5.py (hunk 2 above)
Comment (GatKrieger) [reply ugvc/scripts/sorter_to_h5.py:115]: this part (add the F80, F90, F95 metrics to RawWgsMetrics) is part of the base_coverage condition. if base_coverage is empty, it takes these metrics from the sorter.csv

**Pull request merged**